quickwit/quickwit-search/src/lib.rs

```diff
@@ -165,7 +166,7 @@ fn extract_split_and_footer_offsets(split_metadata: &SplitMetadata) -> SplitIdAn
     }
 }
 
-/// Extract the list of relevant splits for a given search request.
+/// Extract the list of relevant splits for a given query.
```

Comment:
I prefer request here.
We tend to talk about `query` to describe the predicate that selects documents within an index.